src/Utopia/Messaging/Adapters/Push/PushBullet.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\Push;
+
+use Exception;
+
+use Utopia\Messaging\Adapters\Push as PushAdapter;
+use Utopia\Messaging\Messages\Push;
+use GuzzleHttp\Client;
+
+class PushBullet extends PushAdapter
+{
+    /**
+     * @param  string  $apiKey The PushBullet API key.
+     */
+    public function __construct(string $apiKey)
+    {
+        private string $apiKey;
+    }
+
+    /**
+     * Get adapter name.
+     *
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'PushBullet';
+    }
+
+    /**
+     * Get max messages per request.
+     *
+     * @return int
+     */
+    public function getMaxMessagesPerRequest(): int
+    {
+        return 1000;
+    }
```

Comment:
Do you have any reference sources for where you found this number?

Reply:
I didn't find the limit in PushBullet documentation so i just add the example limit